packages/material-ui/src/Grid/Grid.d.ts

```diff
@@ -92,7 +76,26 @@ export type GridClassKey =
   | 'grid-xs-11'
   | 'grid-xs-12';
 
-declare const Grid: React.ComponentType<GridProps>;
+declare const Grid: OverridableComponent<{
+  props: {
+    alignContent?: GridContentAlignment;
+    alignItems?: GridItemsAlignment;
+    container?: boolean;
+    direction?: GridDirection;
+    item?: boolean;
+    justify?: GridJustification;
+    lg?: GridSize;
```

Comment:
I'm not sure about that aspect of the changes. Maybe we should style rely on `Partial<Record<Breakpoint, boolean | GridSize>>`.

Reply:
Ok, changing it.